shared/common-adapters/web-view.native.js

```diff
@@ -15,9 +15,9 @@ const combineJavaScriptAndCSS = (injections?: WebViewInjections) =>
   document.body.appendChild(style)
   style.type = 'text/css'
   style.appendChild(document.createTextNode(\`${escape(injections.css)}\`))
-})()
+})();
```

Comment:
This is the actual fix. Without this, the parenthesis below become a function call. I can't believe I only realized that now!